gum/guminterceptor.c

```diff
@@ -27,6 +27,10 @@
 # define GUM_INTERCEPTOR_CODE_SLICE_SIZE 256
 #endif
 
+#if defined(__x86_64__)
```

Comment:
```suggestion
#if defined (HAVE_I386) && GLIB_SIZEOF_VOID_P == 8
```

(For consistency.)

Should also check that GCC or Clang is used, as MSVC doesn't support the inline assembly syntax.